contrib/lib/src/compression/fairing.rs

```diff
@@ -0,0 +1,150 @@
+//! Automatic response compression.
+//!
+//! See the [`Compression`](compression::fairing::Compression) type for further
+//! details.
+
+use rocket::config::{ConfigError, Value};
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::Rocket;
+use rocket::{Request, Response};
+
+crate use super::context::Context;
+crate use super::CompressionUtils;
+
+/// The Compression type implements brotli and gzip compression for responses in
+/// accordance with the Accept-Encoding header. If accepted, brotli compression
+/// is preferred over gzip.
+///
+/// In the brotli compression mode (using the
+/// [rust-brotli](https://github.com/dropbox/rust-brotli) crate), quality is set
+/// to 2 in order to achieve fast compression with a compression ratio similar
+/// to gzip. When appropriate, brotli's text and font compression modes are
+/// used.
+///
+/// In the gzip compression mode (using the
+/// [flate2](https://github.com/alexcrichton/flate2-rs) crate), quality is set
+/// The Compression type implements brotli and gzip compression for responses in
```

Comment:
This single extraneous line snuck in from somewhere, maybe during a rebase/merge.